deluge/ui/gtkui/torrentdetails.py

```diff
@@ -281,7 +280,7 @@ def generate_menu(self):
         menuitem_list.sort()
 
         for pos, name in menuitem_list:
-            menuitem = gtk.CheckMenuItem(self.translate_tabs[name], True)
+            menuitem = Gtk.CheckMenuItem(self.translate_tabs[name], True)
```

Comment:
breaks the label preference tab being added doesnt give a  error. added as FIXME.